### ISSUE TITLE
SEA: Issue with gun.user().get(...).load() returning .once data instead of .load() data

### ISSUE THREAD

**Issue by 509dave16** [amark/gun#580]
@mhelander @amark 
### TLDR
For my 'lists' node, I get all the key/values just fine. But for the 'items' node, I get back all .once data instead of .load data. The graph does not have children nodes in it for whatever reason.

### Code
This file is where the nuts and bolts of what I'm doing with gun reside: https://github.com/509dave16/random-that/blob/master/src/common/services/list.service.ts
If you wanted to repro you could try running the, client and server:
https://github.com/509dave16/random-that
https://github.com/509dave16/random-that-server

### Scenario with gun.user() where this weird behavior happens
It's hard to tell below in the images, but localStorage has all the items and lists.
But when you look at the root.graph in the loading images. You'll notices that the one 'list' dinner is in there. But none of the children of 'items' are there. Just the 'items' node. Not sure why this is.

!!NOTE!! For whatever reason when one of the 'items' children is put for the first time, the root graph has it. But on refreshes, the root graph all of a sudden doesn't have it there when trying to grab all the 'items' children data.

Data in Local Storage
<img width="1001" alt="screen shot 2018-07-12 at 12 37 59 pm" src="https://user-images.githubusercontent.com/2925882/42655289-872fe10e-85d0-11e8-844d-274f346f76e2.png">

Loading Lists Root Graph Data
<img width="1001" alt="getting_lists_children" src="https://user-images.githubusercontent.com/2925882/42655788-ff4ca34c-85d1-11e8-9330-4572fffdc615.png">

Loading Items Root Graph Data
<img width="1001" alt="getting_items_children" src="https://user-images.githubusercontent.com/2925882/42655796-04cc9a5c-85d2-11e8-8a4f-84ac39c43e93.png">

### Working scenario using just gun
Loading Lists Root Graph Data
<img width="1001" alt="getting_lists_with_just_gun" src="https://user-images.githubusercontent.com/2925882/42656381-f50a347e-85d3-11e8-8e7d-4d9b8d4a1cb2.png">
Loading Items Root Graph Data(the data in the items collection being built is cutoff at the top, but the data is in the graph)
<img width="1001" alt="getting_items_with_just_gun" src="https://user-images.githubusercontent.com/2925882/42656383-f83e70c4-85d3-11e8-8623-3c97e668a8a2.png">

**Comment by 509dave16** [comment]
@amark Figured it out. Very subtle. Missed it completely

To anyone who has encountered this problem. It's from re-authenticating multiple times. Which should be prevented by checking gun.user().is. I had originally done this when rolling my own to start. And when I came back around from SEAs recall I completely forgot about it. If allowed to happen you will get messages from SEA that the data has a different owner. Which is caused by re-authenticating multiple times :| . So you can't get the data if you're not the original owner.
<img width="1010" alt="screen shot 2018-07-12 at 8 41 29 pm" src="https://user-images.githubusercontent.com/2925882/42679476-e71199a8-8636-11e8-96f1-1ee7b8d44266.png">

**Comment by amark** [comment]
@509dave16 thanks. I'm impressed with your thoroughness, hopefully this will be beneficial for people in the future!
